terraform/main.tf

```diff
@@ -2,8 +2,75 @@ provider "aws" {
   region = "eu-west-2" 
 }
 
+# Stuff that exists already
+data "aws_vpc" "c9-vpc" {
+    id = "vpc-04423dbb18410aece"
+}
+
+data "aws_ecs_cluster" "c9-cluster" {
+    cluster_name = "c9-ecs-cluster"
+}
+
+
 # Create ECR repository 
 
+resource "aws_ecr_repository" "dashboard-repo" {
+  name                 = "c9-butterflies-dashboard-terraform"
+  image_tag_mutability = "MUTABLE"
+
+  image_scanning_configuration {
+    scan_on_push = false
+  }
+}
+
+
+# # Create task definition 
+
+resource "aws_ecs_task_definition" "dashboard-taskdef" {
+    family = "c9-butterflies-dashboard-taskdef-terraform"
+    requires_compatibilities = ["FARGATE"]
+    network_mode = "awsvpc"
+    container_definitions = jsonencode([
+    {
+      "name": "c9-butterflies-dashboard-terraform",
+      "image": "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c9-butterflies-dashboard-repo:latest",
+      essential = true
+      environment = [
+        { name = "DB_NAME", value = "${var.DB_NAME}" },
+        { name = "DB_USER", value = "${var.DB_USER}" },
+        { name = "DB_PASSWORD", value = "${var.DB_PASSWORD}" },
+        { name = "DB_PORT", value = "${var.DB_PORT}" },
+        { name = "DB_HOST", value = "${var.DB_HOST}" },
+        { name = "DB_SCHEMA", value = "${var.DB_SCHEMA}" }
+      ]
+      portMappings = [
+        {
+          name          = "c9-butterflies-dashboard-8501-tcp"
+          containerPort = 8501
+          hostPort      = 8501
+          "protocol": "tcp",
+          "appProtocol": "http"
+        }
+      ]
+    }])
+    cpu = 1024
+    memory = 3072
+    execution_role_arn = "arn:aws:iam::129033205317:role/ecsTaskExecutionRole"
+
+}
+
+resource "aws_ecs_service" "dashboard-service" {
+    name = "c9-butterflies-dashboard-service-terraform"
```

Comment:
Just remove the terraform parts like you said, I think Jack is doing this anyway so happy with this